src/processing/vFramer/include/vDraw.h

```diff
@@ -51,10 +51,11 @@ class vDraw {
     unsigned int display_window;
     unsigned int max_window;
     bool flip;
+    bool scaling;
```

Comment:
I don't think the addition of a scale parameter is the best approach. It would be better to automatically scale the y-axis by the largest neuron id you have read so far.